### ISSUE TITLE
would you please support more linespecs

### ISSUE THREAD

**Issue by skyblueee** [Profactor/cv-plot#5]
currently there are only solid linestyle and circle/dot pointstyle supplied.
Maybe at least solidline/dotline, circle/dot/star/*/+ supported?
colors are enough now I think.

**Comment by wpalfi** [comment]
Good point. I would like to rewrite the Series implementation anyway, but I dont have time right now. May take a few weeks.
If anybody has a nice implementation in the meantime, please share it!

**Comment by sharowyeh** [comment]
I recently got into this repo and added cross and plus marker types in my fork commit 7e595651f01ccd2e1afb2711f9097b076c40d9a9 for personal usage.
It may be helpful

**Comment by AbsurdePhoton** [comment]
Thanks to your commit [sharowyeh](https://github.com/sharowyeh), I could also understand how to add these shapes : square, diamond (tilted square), triangle up, and triangle down. Quite easy to add/draw !
